libs/infra-next-server/src/lib/health.ts

```diff
@@ -64,6 +64,6 @@ export const setupHealthchecks = (
 
   app.use('/liveness', (_, res) => {
     res.statusCode = 200
-    res.end('')
+    res.json({ ok: true })
```

Comment:
Should make minimal changes
```suggestion
    res.end('')
```